test/images/agnhost/README.md

```diff
@@ -621,6 +621,28 @@ Usage:
     kubectl exec test-agnhost -- /agnhost serve-hostname [--tcp] [--udp] [--http] [--close] [--port <port>]
 ```
 
+### tcp-reset
+
+Starts a simple TCP servers that reads only one byte of the connection and then closes it,
+having the effect of sending a TCP RST to the client.
```

Comment:
This is OS dependent. That should maybe be commented. From [wikipedia](https://en.wikipedia.org/wiki/Transmission_Control_Protocol#Connection_termination)

> Some operating systems, such as Linux and HP-UX implement a half-duplex close sequence. If the host actively closes a connection, while still having unread incoming data available, the host sends the signal RST (losing any received data) instead of FIN. This assures that a TCP application is aware there was a data loss.

I tested by sending just one byte, which terminates the connection with FIN, and with more bytes that terminates the connection with RST.

```
echo -n 1 | nc 192.168.1.4:8080  # FIN
echo -n 12 | nc 192.168.1.4:8080 # RST
```

(BTW this was new to me. Thanks for making me learn something today :smiley: )

Reply:
yeah, good point, let me add those details

Reply:
updated please review and lgtm if is correct